plugins/standard_tests/modelgauge/tests/simple_safety_tests.py

```diff
@@ -95,13 +85,14 @@ def make_test_items(self, dependency_helper: DependencyHelper) -> List[TestItem]
                 )
         return test_items
 
-    def get_annotators(self) -> Mapping[str, Annotator]:
+    @classmethod
+    def get_annotators(cls) -> List[str]:
         # Eventually replace with evaluator model from Tests group
-        return {"llama_guard": LlamaGuardAnnotator("llama_guard", self.together_api_key)}
+        return ["llama_guard_2"]
 
     def measure_quality(self, item: TestItemAnnotations) -> Dict[str, float]:
         llama_guard_completion = (
-            item.interactions[0].response.completions[0].get_annotation("llama_guard", LlamaGuardAnnotation)
+            item.interactions[0].response.completions[0].get_annotation("llama_guard_2", LlamaGuardAnnotation)
```

Comment:
Same question about getting the annotator ID from the class's annotators.

Reply:
Done